src/OperationResponse.php

```diff
@@ -408,9 +414,6 @@ public function getMetadata()
 
     private function operationsCall($method, $name, array $additionalArgs)
     {
-        if (is_null($method)) {
```

Comment:
Why remove this? There is a "new" seg fault in the unit tests, could this be why?

Reply:
I didn't remove it, I just moved it into the two methods where we think it could happen.